server/controllers/users.js

```diff
@@ -0,0 +1,124 @@
+import passport from 'passport';
+import models from '../models';
+
+const { User } = models;
+
+/**
+ * @exports UserController
+ * @class UserController
+ * @description Handles Social Users
+ * */
+class UserController {
+  /**
+  * Create a new user
+  * @async
+  * @param  {object} req - Request object
+  * @param {object} res - Response object
+  * @param {object} next The next middleware
+  * @return {json} Returns json object
+  * @static
+  */
+  static async create(req, res, next) {
+    const { username, email, password } = req.body;
+
+    try {
+      const user = await User.create({ username, email, password });
+      return res.status(201).send({ user });
+    } catch (error) {
+      next(error);
+    }
+  }
+
+  /**
+  * Login and Authenticate user.
+  * @async
+  * @param  {object} req - Request object
+  * @param {object} res - Response object
+  * @param {object} next The next middleware
+  * @return {json} Returns json object
+  * @static
+  */
+  static async login(req, res, next) {
+    if (!req.body.email) {
```

Comment:
consider using/creating some sort of validator for this.

would be nice/it is possible to have these multiple validations done at once (validate email, password etc)